hikari/impl/rest.py

```diff
@@ -2137,9 +2138,98 @@ async def delete_emoji(
         self,
         guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
         emoji: snowflakes.SnowflakeishOr[emojis.CustomEmoji],
+        *,
+        reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> None:
         route = routes.DELETE_GUILD_EMOJI.compile(guild=guild, emoji=emoji)
-        await self._request(route)
+        await self._request(route, reason=reason)
+
+    async def fetch_available_sticker_packs(self) -> typing.Sequence[stickers.StickerPack]:
+        route = routes.GET_STICKER_PACKS.compile()
+        response = await self._request(route)
```

Comment:
test coverage for all additions in this module?

Reply:
Test coverage is still something I got to do, havent had time too